### ISSUE TITLE
BROKEN: puppetteer example is broken with error: spawn EACCES

### ISSUE THREAD

**Issue by elie222** [zeit/now-examples#361]
Steps to reproduce:

```sh
now init # select `puppeteer-screenshot`
cd puppeteer-screenshot
now dev
```

Then open `http://localhost:3000/` in your browser.
Click any of the links.

You'll see this error on screen:

```
Server Error
Sorry, there was a problem
```

And this in the server console: `spawn EACCES`.

**Comment by tajo** [comment]
Yea. It works remotely with `now` but not with `now dev`. The error output:

```sh
~/Projects/now-examples/puppeteer-screenshot: master % u= now dev
> Now CLI 15.2.0 dev (beta) — https://zeit.co/feedback/dev
> Ready! Available at http://localhost:3000
> GET /
> GET /google.com
> Building @now/node:screenshot.js
downloading user files...
installing dependencies for user's code...
installing to /var/folders/fz/2bk2f3_n22993xrkknthd7yr0000gn/T/co.zeit.now/dev/workPaths/hlvtgg58
running user script...
compiling entrypoint with ncc...
ncc: Version 0.18.5
ncc: Compiling file index.js
ncc: Module directory "/private/var/folders/fz/2bk2f3_n22993xrkknthd7yr0000gn/T/co.zeit.now/dev/workPaths/hlvtgg58/node_modules/chrome-aws-lambda/source" attempted to require "puppeteer" but could not be resolved, assuming external.
ncc: Module directory "/private/var/folders/fz/2bk2f3_n22993xrkknthd7yr0000gn/T/co.zeit.now/dev/workPaths/hlvtgg58/node_modules/chrome-aws-lambda/source/puppeteer/lib" attempted to require "puppeteer" but could not be resolved, assuming external.
ncc: Module directory "/private/var/folders/fz/2bk2f3_n22993xrkknthd7yr0000gn/T/co.zeit.now/dev/workPaths/hlvtgg58/node_modules/chrome-aws-lambda/source/puppeteer/lib" attempted to require "puppeteer" but could not be resolved, assuming external.
ncc: Module directory "/private/var/folders/fz/2bk2f3_n22993xrkknthd7yr0000gn/T/co.zeit.now/dev/workPaths/hlvtgg58/node_modules/debug/src" attempted to require "supports-color" but could not be resolved, assuming external.
ncc: Module directory "/private/var/folders/fz/2bk2f3_n22993xrkknthd7yr0000gn/T/co.zeit.now/dev/workPaths/hlvtgg58/node_modules/https-proxy-agent/node_modules/debug/src" attempted to require "supports-color" but could not be resolved, assuming external.
ncc: Module directory "/private/var/folders/fz/2bk2f3_n22993xrkknthd7yr0000gn/T/co.zeit.now/dev/workPaths/hlvtgg58/node_modules/ws/lib" attempted to require "bufferutil" but could not be resolved, assuming external.
ncc: Module directory "/private/var/folders/fz/2bk2f3_n22993xrkknthd7yr0000gn/T/co.zeit.now/dev/workPaths/hlvtgg58/node_modules/ws/lib" attempted to require "utf-8-validate" but could not be resolved, assuming external.
> Built @now/node:screenshot.js [11s]
dlopen(/private/var/folders/fz/2bk2f3_n22993xrkknthd7yr0000gn/T/zeit-fun-abf206e874455/source/iltorb/build/bindings/iltorb.node, 1): no suitable image found.  Did find:
        /private/var/folders/fz/2bk2f3_n22993xrkknthd7yr0000gn/T/zeit-fun-abf206e874455/source/iltorb/build/bindings/iltorb.node: unknown file type, first eight bytes: 0x7F 0x45 0x4C 0x46 0x02 0x01 0x01 0x00
        /private/var/folders/fz/2bk2f3_n22993xrkknthd7yr0000gn/T/zeit-fun-abf206e874455/source/iltorb/build/bindings/iltorb.node: unknown file type, first eight bytes: 0x7F 0x45 0x4C 0x46 0x02 0x01 0x01 0x00
^C
> Stopping `now dev` server
```

It seems that `puppeteer` can't be found. Maybe some ENV variable I am missing?

**Comment by bborn** [comment]
Same. Looks like it isn't finding the required dependencies.

**Comment by daryl** [comment]
Also running into this, any news?

**Comment by leo** [comment]
Thanks for reporting this! Would you mind replacing `@now/node` with `@now/node@canary`? I believe it will fix the problem.

**Comment by daryl** [comment]
That's what I'm using currently, I've tried `@now/node` and `@now/node@canary`, I still get an EACCESS error for `/tmp/chromium`.

```
Error: spawn /tmp/chromium EACCES
    at Process.ChildProcess._handle.onexit (internal/child_process.js:247:19)
    at onErrorNT (internal/child_process.js:431:16)
    at processTicksAndRejections (internal/process/task_queues.js:81:17)
LambdaError: RequestId: b4110c86-4dd5-4f6e-b12a-24dc63f58745 Process exited before completing request
    at Lambda.<anonymous> (/snapshot/repo/dist/index.js:1:2542456)
    at Generator.next (<anonymous>)
    at fulfilled (/snapshot/repo/dist/index.js:1:2540597)
    at process._tickCallback (internal/process/next_tick.js:68:7)
```

**Comment by styfle** [comment]
I submitted a fix in PR #417, thanks!

**Comment by msweeneydev** [comment]
@tajo, @elie222, @bborn - this has now been fixed!
